tox.ini

```diff
@@ -4,7 +4,7 @@
 # and then run "tox" from this directory.
 
 [tox]
-testenv=py{279,36,275}
+testenv=py{36,38,39,310}
 
 [base]
 deps =
```

Comment:
- maybe `unittest2` and `requests` can be removed now.

Reply:
I don't get what this means, are you suggest remove the tox config and  tier_tests.py?

Reply:
Got it, thanks